lib/board.rb

```diff
@@ -32,8 +37,45 @@ def free_positions
     flat_board.each_index.select { |index| empty?(flat_board[index]) }
   end
 
+  def content
```

Comment:
[[x,o,_],[_,_,_],[x,x,o]] --> String --> "xo ,   ,xxo"

So this is what the game will return to the cli/web/...
If I have to change my double array datastructure to another one it will still return the same kind of string which will no break the projects using this class